api/openapi-spec/swagger.json

```diff
@@ -15617,6 +15619,7 @@
       "$ref": "#/definitions/v1.LogVerbosity"
      },
      "memoryOvercommit": {
+      "description": "MemoryOvercommit is the percentage of memory we want to give VMIs compared to the amount given to its parent pod (virt-launcher). For example, a value of 102 means the VMI will \"see\" 2% more memory than its parent pod. Values under 100 are effectively \"undercommits\". Defaults to 100",
```

Comment:
Is it worth (briefly) mentioning potential downsides of overcommit?

Reply:
Added. PTAL!